alpine-nix-rails-nginx/etc/nginx/static-content.conf

```diff
@@ -0,0 +1,42 @@
+# Default static content to be cached for 1 day
+expires 1d;
+
+if ($request_method !~ ^(GET|HEAD|OPTIONS)$) {
+  more_set_headers 'Allow: GET,HEAD,OPTIONS';
+  return 405;
+}
+
+if ($request_method = OPTIONS) {
+  more_set_headers 'Allow: GET,HEAD,OPTIONS';
+  return 204;
+}
+
+# Prefer the static files before proxying to the app
+try_files $uri @rails;
+
+location = /robots.txt  {
+  access_log off;
```

Comment:
I think that this is a bad setting. We cannot declare a particular URL as "save and we never need insights into its access pattern". Potentially there can be any bug in nginx (also in future additions / plugins etc) that can be exploited.

Imagine an exploit that needs a million requests to succeed. An attacker would choose a fast target (robots.txt) and could also assume "nobody logs this requests" so our monitoring would not even see a spike warning us. Also for postmortems reducing visibility, even on "currently innocent looking" URLs could be wasteful.

If logging these requests becomes a size problem, we'll have the resources to compensate.